rdt-open-reader/src/main/java/io/ona/rdt/fragment/RDTJsonFormFragment.java

```diff
@@ -229,4 +239,39 @@ public View getRootLayout() {
     public RDTJsonFormActivity getRdtActivity() {
         return (RDTJsonFormActivity) getActivity();
     }
+
+    @Override
+    public void writeValue(String stepName, String prentKey, String childObjectKey, String childKey, String value, String openMrsEntityParent, String openMrsEntity, String openMrsEntityId, boolean popup) {
+        super.writeValue(stepName, prentKey, childObjectKey, childKey, value, openMrsEntityParent, openMrsEntity, openMrsEntityId, popup);
+        if (STEP_8.equals(stepName) && CovidConstants.FormFields.PATIENT_DETAIL.equals(prentKey) && AllConstants.OPTIONS.equals(childObjectKey)) {
+            setNextButtonState(rootLayout.findViewById(com.vijay.jsonwizard.R.id.next_button), isPatientInfoConfirmed(stepName));
+        }
+    }
+
+    private boolean isPatientInfoConfirmed(String currentStep) {
```

Comment:
To prevent navigation when the checkboxes are not checked, add `required` constraint on the checkbox widgets.

We don't need to explicitly disable the button.

Reply:
added `v_required` keyword